src/routes/movieRouter.js

```diff
@@ -5,20 +5,117 @@ import {
     updateTestMovie,
 } from '../test-values/movieTestValues.js'
 import { db } from '../db/db.js'
+import slugify from 'slugify'
 
 const movieRouter = Router()
 
+//get all movies
 movieRouter.get('/', async (req, res) => {
-    const result = await db.query('SELECT * FROM movies')
+    const result = await db.query(
+        'SELECT m.slug, m.title, m.description, m.img FROM movies AS m'
```

Comment:
```suggestion
        'SELECT m.slug, m.title, m.description, m.date_added, m.img FROM movies AS m'
```

Had date added down as a return variable in the API. We haven't really spoken about it so I'll message you!